source/less/blocks/extra.less

```
@@ -0,0 +1,15 @@
+.extra__text{
```

```suggestion
.extra__text {
```

Пробелы везде нужны. А то где-то есть, а где-то нет.